packages/@vue/cli/bin/vue.js

```diff
@@ -261,7 +258,9 @@ if (!process.argv.slice(2).length) {
 function suggestCommands (unknownCommand) {
   const availableCommands = program.commands.map(cmd => cmd._name)
 
-  const suggestion = didYouMean(unknownCommand, availableCommands)
+  const suggestion = availableCommands.find(cmd => {
+    return leven(cmd, unknownCommand) < 3
```

Comment:
Better sort and find the closest result.
Otherwise, it might return `init` for the `infi` (should return `info`).

Reply:
Could you elaborate?

Reply:
Using `find` it will suggest the first command that is less than 3 characters away from the input. It may not be the best guess because there may be a command that is even closer to the user input.

Reply:
Gotcha.

Reply:
Do you have any suggestion regarding the implementation? we've to iterate through the entire array of available commands and find the most appropriate match. One approach that I could think of is to filter out the matches and going ahead picking up the right one from them.

Reply:
Just iterate the array and find the one with the shortest distance would suffice.